pkg/acsengine/defaults.go

```diff
@@ -98,8 +98,10 @@ func setDefaultCerts(a *api.Properties) (bool, error) {
 
 	ips := []net.IP{firstMasterIP}
 
-	for i := 1; i < a.MasterProfile.Count; i++ {
-		ips = append(ips, net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)})
+	// Include the Internal load balancer as well
+	for i := 1; i < (a.MasterProfile.Count + 1); i++ {
+		ip := net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)}
```

Comment:
Okay. So, my guess is... the code that constructs `firstMasterIP` is using `IPv4()` constructor which puts the ipv4 octets in 12,13,14,15.

So, that's what we should do here. Can you change it to use the IPv4 constructor? (I still have no idea how this is working now I guess...)